docs/plugin-maintainers-guide.md

```diff
@@ -0,0 +1,38 @@
+# Plugin Maintainer Guide
+
+## Table of Contents
+
+- [Plugin Maintainer Guide](#plugin-maintainer-guide)
+  - [Table of Contents](#table-of-contents)
+  - [Maintaining and patching an older release line](#maintaining-and-patching-an-older-release-line)
+    - [Patching an older release](#patching-an-older-release)
+
+## Maintaining and patching an older release line
+
+It may be necessary to patch a prior release line of a plugin when users depend on an older, but stable version and while a newer, incompatible, major version of the plugin exists. Typically for these older releases, only major bugs and security issues will need to be remediated. Not every plugin will need this workflow.
+
+This guide will describe the steps needed to release on an older version.
+
+### Patching an older release
+
+When patching an older release, follow the steps below to ensure the correct workflow is applied:
+
+1. Request a `workspace/${workspace}` branch:
```

Comment:
How does one request a branch?

Reply:
if you need to be an org member to run the workflow, then you might as well ping that person at the outset to get them to create the branch so you can then later run the workflow. 

An issue template would boilerplate the request and give people something to link to for the branch creation and later workflow runs too, so maybe that's worth doing.